tests/sequence/test_fasta.py

```diff
@@ -40,22 +40,28 @@ def test_access_low_level():
 
 
 def test_access_high_level():
-    path = os.path.join(data_dir("sequence"), "nuc.fasta")
-    file = fasta.FastaFile.read(path)
-    sequences = fasta.get_sequences(file)
-    assert sequences == {
+    sequences_reference = {
         "dna sequence" : seq.NucleotideSequence("ACGCTACGT", False),
         "another dna sequence" : seq.NucleotideSequence("A", False),
         "third dna sequence" : seq.NucleotideSequence("ACGT", False),
         "rna sequence" : seq.NucleotideSequence("ACGT", False),
         "ambiguous rna sequence" : seq.NucleotideSequence("ACGTNN", True),
     }
+    path = os.path.join(data_dir("sequence"), "nuc.fasta")
+    file = fasta.FastaFile.read(path)
+    sequences = fasta.get_sequences(file)
+    assert sequences == sequences_reference
+    sequences_manual = fasta.get_sequences(file, seq.NucleotideSequence)
+    assert sequences_manual == sequences_reference    
```

Comment:
This this is almost a duplicate of the two lines above, I would suggest using `pytest.mark.parametrize()`to add one version of the test with and one version without manual sequence type selection.